inspect_image.go

```diff
@@ -39,6 +40,8 @@ const (
 	CNBProcessEnv       = "CNB_PROCESS_TYPE"
 	defaultProcess      = "web"
 	fallbackPlatformAPI = "0.3"
+	LauncherEntrypoint  = "/cnb/lifecycle/launcher"
+	BpEntrypointPrefix  = "/cnb/process/"
```

Comment:
We'll need different variables for Windows images.

Reply:
Meaning, in Windows images, it would be `\cnb\lifecycle\launcher` and `\cnb\process`?